packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -48,6 +48,7 @@ import {
   addDepositToCache,
   getProvider
 } from '../components/TransactionHistory/helpers'
+import { useDestinationAddressStore } from '../components/TransferPanel/AdvancedSettings'
```

Comment:
If we reuse `useDestinationAddressStore`, I'd move it to its own store file

Reply:
I would keep any refactoring in TransferPanel-related files, out of the scope of the current PR because:
1. It might conflict with reskin if we plan to launch this earlier.
2. There are a series of refacoring PRs already planned after reskin so it would be better to accommodate this chance there.
I can surely create a ticket for this for now: https://linear.app/offchain-labs/issue/FS-322/refactor-usedestinationaddressstore-to-its-own-file